Dockerfile

```diff
@@ -39,6 +39,9 @@ RUN ln -s /toolchain/etc/ssl/certs/ca-certificates /etc/ssl/certs/ca-certificate
 # fhs
 COPY hack/scripts/fhs.sh /bin
 RUN fhs.sh /rootfs
+# ca-certificates
```

Comment:
We need valid certs in `initramfs` in order to download artifacts in early boot.